settings.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
```

Comment:
This looks too polished to be a one-off script hacked together by you... where did it come from? We should attribute it, no?